src/Surfnet/StepupMiddleware/ApiBundle/Doctrine/Type/UseRaLocationsOptionType.php

```diff
@@ -61,14 +61,20 @@ public function convertToPHPValue($value, AbstractPlatform $platform)
             return $value;
         }
 
+        // Prevent a too lenient boolean conversion on non-postgresql platforms
+        if ($platform->getName() !== 'postgresql') {
+            if ($value !== 1 && $value !== 0 && $value !== '1' && $value !== '0' && !is_bool($value)) {
```

Comment:
Alright, I think `$platform->getFromBoolean($value)` would suffice. Given that we only speak to the database through the DBAL, this should not give any issues.